deploy/crds/saas.3scale.net_apicasts_crd.yaml

```diff
@@ -0,0 +1,353 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: apicasts.saas.3scale.net
+spec:
+  group: saas.3scale.net
+  names:
+    kind: Apicast
+    listKind: ApicastList
+    plural: apicasts
+    singular: apicast
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      type: object
+      description: Apicast is the Schema for Apicast instance
+      properties:
+        apiVersion:
+          description:
+            "APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources"
+          type: string
+        kind:
+          description:
+            "Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds"
+          type: string
+        metadata:
+          type: object
+        spec:
+          type: object
+          description: ApicastSpec defines the desired state of Apicast
+          required:
+            - production
+            - staging
+          properties:
+            production:
+              type: object
+              required:
+                - image
+                - env
+                - marin3r
+                - externalDnsHostname
+              properties:
+                externalDnsHostname:
+                  type: string
+                  description: DNS hostname to manage on AWS Route53 by external-dns
+                image:
+                  type: object
+                  required:
+                    - tag
+                  properties:
+                    name:
+                      type: string
+                      description: Image name (docker repository)
+                    tag:
+                      type: string
+                      description: Image tag
+                    pullSecretName:
+                      type: string
+                      description: Quay pull secret for private repository
+                replicas:
+                  type: integer
+                  description: Number of replicas
+                env:
+                  type: object
+                  description: Environment variables
+                  required:
+                    - apicastConfigurationCache
+                    - threescalePortalEndpoint
+                  properties:
+                    apicastConfigurationCache:
+                      type: string
```

Comment:
Being a field representing time in seconds, shouldnt it be an integer, instead of a string? (applies to examples, docs, CRD)

Reply:
Validations added in 1d9096289bf609153e378ba6ae025774854cbc4f